docs/website/mtouch-errors.md

```diff
@@ -1787,6 +1787,12 @@ Mixed-mode assemblies can not be processed by the linker.
 
 See https://msdn.microsoft.com/en-us/library/x0w2664k.aspx for more information on mixed-mode assemblies.
 
+### MT2105: The method {method} contains a '{handlerType}' exception clause, which is currently not supported when compiling for bitcode. This method will throw an exception if called.
+
+Currently Xamarin.iOS does not support the 'filter' exception clauses when
+compiling to bitcode. Any methods containing such code will throw a
+NotSupportedException exception.
```

Comment:
The description is not clear **when** the exception is thrown.

Is the current behaviour to assert when executing the method ? or 
only when an exception occurs ? or
only when an exception with a filter occurs ?

Reply:
The current behavior is to throw an exception when the method is executed (LLVM completely fails for the method if it contains a filter clause, which means that there's no native code for the method at all). I've seen asserts too, but I'm not sure when it asserts and when an exception is thrown.

I've updated the text to be more descriptive.